pjmedia/src/pjmedia/stream.c

```diff
@@ -2281,6 +2281,7 @@ static void on_rx_rtp( pjmedia_tp_cb_param *param)
         nlost = PJ_MIN(seq_st.diff - 1, 17);
         first_seq = pj_ntohs(hdr->seq) - nlost;
 
+        int i;
```

Comment:
in C we need either to move declaration to the top to improve compatibility with compilers(it's c++ feature to be able to declare variables at any line in func and not at top) or we could declare "i" in for loop
for (int i = 1; i < (seq_st.diff - 1); ++i) {